app/component/front-page/FrontPagePanel.js

```diff
@@ -162,14 +164,14 @@ export default class FrontPagePanel extends React.Component {
         {this.getSelectedPanel() ? content : undefined}
         </ReactCSSTransitionGroup>
         <ul className="tabs-row tabs-arrow-up cursor-pointer">
-          <li className={cx(tabClasses, nearbyClasses)} onClick={clickNearby}>
-            <Icon className="prefix-icon" img="icon-icon_bus-withoutBox" />
-            <FormattedMessage id="near-you" defaultMessage="Near you" />
-          </li>
-          <li className={cx(tabClasses, favouritesClasses)} onClick={clickFavourites}>
-            <Icon className="prefix-icon" img="icon-icon_star" />
-            <FormattedMessage id="your-favourites" defaultMessage="Your favourites" />
-          </li>
+          <NearbyTabLabelContainer
+            classes={cx(tabClasses, nearbyClasses)}
```

Comment:
Can we inline these classes to FavouritesTabLabel and NearbyTabLabel as they are statically defined